js/classes/animator.js

```diff
@@ -2,7 +2,7 @@
  * This class is for animating a spritesheet.
  * An Animator is responsible for only ONE of the sprites Animations (i.e. jumping up, walking right, etc.),
  * therefore many sprites will have several Animators.
- * @author Chris Marriott, Devin Peevy
+ * @author Chris Marriott, Devin Peevy, Nathan Hinthorne
```

Comment:
I think multiple authors are usually listed on separate lines with JSDoc.

Reply:
Good catch! I forgot about that. I'll go fix it